pkg/tsdbctl/query.go

```diff
@@ -113,6 +114,8 @@ Arguments:
 		"Aggregation interval for applying the aggregation functions\n(if set - see the -a|--aggregates flag), of the format\n\"[0-9]+[mhd]\" (where 'm' = minutes, 'h' = hours, and\n'd' = days). Examples: \"1h\"; \"150m\". (default =\n<end time> - <start time>)")
 	cmd.Flags().StringVar(&commandeer.groupBy, "groupBy", "",
 		"Comma separated list of labels to group the result by")
+	cmd.Flags().BoolVar(&commandeer.usePreciseAggregations, "use-precise-aggregations", false,
+		"Ignore server aggregations optimizations for more accurate results.")
```

Comment:
`Ignore server aggregations optimizations` is not correct English :-). You can change `aggregations` -> `aggregation`, drop the `optimizations `, or rephrase.